static/docs/commands-reference/commit.md

```diff
@@ -2,40 +2,212 @@
+    $ cd example-get-started
+    $ dvc pull
+```
+
+The workspace in the Git repository is preconfigured with a DVC remote containing
+precomputed data we can use in the following examples.
+
+## Example: Editing a data file
+
+Sometimes we want to edit a data or configuration file and rerun the pipeline to
+see what difference it makes.  To replace a data file we'd use the `dvc remove`
+command, but to edit it we use `dvc unprotect`.
+
+```dvc
+    $ ls -l data/
+    total 74072
+    -rw-r--r--  2 david  admin  37916850 Apr 11 17:03 data.xml
+
+    $ dvc unprotect data/data.xml
+    [##############################] 100% Unprotecting 'data/data.xml'
+
+    $ ls -l data/
+    total 74072
+    -rw-r--r--  1 david  admin  37916850 Apr 11 17:06 data.xml
+```
+
+The `1` in link count here indicates DVC has unprotected the file.  For other
+link types this will appear in different ways.  At this point we can safely edit
+the data file.
+
+After editing the file we can use `dvc commit` to update the changes.
+
+```dvc
+    $ dvc commit
+    ...
+    outputs ['data/data.xml'] of 'data/data.xml.dvc' changed. Are you sure you commit it? [y/n] y
+    ...
+    dependencies ['data/data.xml'] of 'prepare.dvc' changed. Are you sure you commit it? [y/n] y
+```
+
+Because this file is referenced from two DVC files, we are queried twice.
+
+## Example: Running the pipeline without committing data changes
+
+As was suggested in the previous section, sometimes we want to change the data
+to try different options.  To avoid filling the cache with temporary results as
+you try different algorithms we can unprotect all affected files, and use the
+`dvc repro --no-commit` option to prevent the cache from being updated.
+
+We start with unprotecting the files involved in the affected stages:
+
+```dvc
+    $ dvc unprotect data/data.xml data/prepared/ data/features/ model.pkl 
+```
+
+We can now edit `data/data.xml` or any file in the `src` directory.
+
+To rerun the pipeline without committing data to the cache:
+
+```dvc
+    $ dvc repro --no-commit train.dvc 
+```
+
+And we might end up with a status like this:
+
 ```dvc
-    $ echo foo > foo
-    $ dvc add foo
-    $ dvc run -d foo -o bar --no-commit 'echo bar > bar'
     $ dvc status
-    bar.dvc:
-            changed outs:
-                    missing cache:      bar
+
+    data/data.xml.dvc:
+        changed outs:
+            not in cache:       data/data.xml
+    evaluate.dvc:
+        changed deps:
+            modified:           data/features
+            modified:           model.pkl
+    train.dvc:
+        changed outs:
+            not in cache:       model.pkl
+```
+
+Once we're satisfied with the changes, they can be committed to the DVC cache:
+
+```dvc
     $ dvc commit
+    ... commit output
     $ dvc status
     Pipeline is up to date. Nothing to reproduce.
 ```
+
+## Example: Execute a stage on a remote machine
+
+Sometimes code must be executed somewhere else, for example on an HPC cluster or
```

Comment:
It's a pretty artificial example - no need to move the artifact manually and replace it. The recommended way is to push to the remote and pull the result locally. I would remove this section. Let's focus on the previous one. Specifically show that, for example we want to try different number of features (see how we change 5000 to 6000 in the get started). And we are trying, and trying ... and we don't want to keep the artifacts in cache. Even better there is a git tag that represents the "baseline" iteration. You can say that it's often we iterate on top of baseline before we actually get the right values and are ready to do the "bigrams" commit, show multiple iterations in `--no-commit` and then do the final `dvc commit` + `git commit`.

Reply:
Removed